mm2src/mm2_main/src/lp_swap/maker_swap_v2.rs

```diff
@@ -424,8 +424,12 @@ impl<MakerCoin: MmCoin + MakerCoinSwapOpsV2, TakerCoin: MmCoin + TakerCoinSwapOp
     #[inline]
     fn unique_data(&self) -> Vec<u8> { self.secret_hash() }
 
-    fn dex_fee(&self, taker_pub: Option<&[u8]>) -> DexFee {
-        DexFee::new_from_taker_coin(
+    fn dex_fee_max(&self) -> DexFee {
```

Comment:
`dex_fee_max` is a very confusing name for me.

Reply:
I created this function to handle optionality of the taker_pub.
(Above I suggested to rethink that and maybe handle this inside the DexFee::new_from_taker_coin fn)

Reply:
I see that the maker will not be aware of the taker htlc pubkey at this stage, only the p2p key https://github.com/KomodoPlatform/komodo-defi-framework/blob/74202d36facfaa1681894c4ba11d91209e505fe7/mm2src/mm2_main/src/lp_swap/maker_swap_v2.rs#L404-L405 In the future the p2p key will probably be the same as the htlc key for kyc/aml purposes. In the meantime, it's only the function name that is confusing, so you can change it or reverse the splitting of `new_from_taker_coin` into 2 functions. Whatever you prefer as I am sure this code will be changed later.

Reply:
I renamed both functions in [87a1e49](https://github.com/KomodoPlatform/komodo-defi-framework/pull/2112/commits/87a1e498a184a826b4c6c1ded618578746c31a3d)